content/knowledge-base/migrating-from-bitrise.md

```diff
@@ -0,0 +1,202 @@
+
+If you have already set up manual code signing on Bitrise, setting it up with Codemagic is relatively simple. You can download the necessary provisioning profiles and certificates from Bitrise's `Code Signing` tab. To add them as environment variables in Codemagic, navigate to your application and click the `Environment variables` tab. Encode the contents of your files with base64 and paste the values into Codemagic; make sure to check `Secure` when providing sensitive information. Along with creating these variables, create a new variable group, e.g., `ios_code_signing`, which you can later reference in your `codemagic.yaml`. For convenience, we recommend naming your variables as `FCI_CERTIFICATE`, `FCI_CERTIFICATE_PASSWORD`, and `FCI_PROVISIONING_PROFILE`. However, you can call the variables differently if you reference them correctly in the scripts section. In the case of multiple provisioning profiles, the recommended naming convention is `FCI_PROVISIONING_PROFILE_1`, `FCI_PROVISIONING_PROFILE_2` etc.
+
+As mentioned above, unlike with the Bitrise `Certificate and profile installer` step, you must reference the added files in your scripts section. Follow the detailed documentation [here](../yaml-code-signing/signing-ios/#manual-code-signing).
+
+#### Automatic
+
+Codemagic uses the [App Store Connect API](../yaml-code-signing/signing-ios/#creating-the-app-store-connect-api-key) for managing signing profiles and certificates. Bitrise uses the same API in the `Manage iOS Code Signing` step.
+
+Thus, Codemagic requires the same values as Bitrise for handling automatic code signing. With the `codemagic.yaml` configuration, the values have to be provided as [environment variables](../yaml-code-signing/signing-ios/#saving-the-api-key-to-environment-variables). In addition, Codemagic requires you to create an RSA 2048 bit private key to be included in the signing certificate. 
+
+Then, using the defined variables in your scripts section to automate creating and fetching profiles and certificates is possible.
+
+Follow the steps defined in our [documentation](../yaml-code-signing/signing-ios/#automatic-code-signing) to find information on how to generate the necessary details and use them in your configuration.
+
+### Building
+
+In Bitrise, the archiving to create the IPA for distribution is done in the `Xcode Archive & Export for iOS` step. 
+
+Suppose you have followed the above instructions on setting up code signing with the `codemagic.yaml` configuration. In that case, to continue with Codemagic, add a script to your scripts sections to build the application. You can find various build scripts for different types of applications under our [Quick start guides](../yaml-quick-start/codemagic-sample-projects/).
+
+Likewise, unlike Bitrise's `Deploy to Bitrise.io` step, with Codemagic, you have to configure all the wanted artifacts under the [artifacts section](../yaml/yaml-getting-started/#artifacts) in your `codemagic.yaml` configuration.
```

Comment:
Let's change emphasis in this section. E.g. With Codemagic, you can configure which artifacts to receive at the end of the build (add an example of all the relevant one for iOS, xcode log, dsym file, etc.). Also, mention that build artifacts for teams do not expire.